app/src/main/java/org/stepic/droid/ui/adapters/CoursesAdapter.java

```diff
@@ -140,6 +155,28 @@ private void onClickCourse(int position) {
         }
     }
 
+    private void showMore(View view, @NotNull final Course course) {
+        PopupMenu morePopupMenu = new PopupMenu(contextActivity, view);
+        morePopupMenu.inflate(ContextMenuCourseUtil.INSTANCE.getMenuResource(course));
+
+        morePopupMenu.setOnMenuItemClickListener(new PopupMenu.OnMenuItemClickListener() {
+            @Override
+            public boolean onMenuItemClick(MenuItem item) {
+                switch (item.getItemId()) {
+                    case R.id.menu_item_info:
+                        screenManager.showCourseDescription(contextActivity, course);
+                        return true;
+                    case R.id.menu_item_unroll:
+                        droppingPresenter.dropCourse(course);
```

Comment:
Maybe we should show some kind of dialog where user have to confirm his will to drop course for preventing accidental clicks?

Reply:
If we will observe strange drop-rate, we can add it. See details [here](https://vyahhi.myjetbrains.com/youtrack/issue/APPS-1370#comment=74-38639).